jib-cli/src/main/java/com/google/cloud/tools/jib/cli/cli2/Jar.java

```diff
@@ -43,6 +44,14 @@
   @SuppressWarnings("NullAway.Init") // initialized by picocli
   private Path jarFile;
 
+  @CommandLine.Option(
+      names = "--mode",
+      defaultValue = "exploded",
+      paramLabel = "<mode>",
+      description = "The processing mode")
```

Comment:
I think this will be what the user will see on the command-line with the help usage. Should include what values are available.

That said, take a look at the `--vebosity` and `--console` options.

```java
  @Option(
      names = "--verbosity",
      paramLabel = "<level>",
      defaultValue = "lifecycle",
      description =
          "set logging verbosity, candidates: ${COMPLETION-CANDIDATES}, default: ${DEFAULT-VALUE}")
  @SuppressWarnings("NullAway.Init") // initialized by picocli
  private Verbosity verbosity;

  @Option(
      names = "--console",
      paramLabel = "<type>",
      defaultValue = "auto",
      description =
          "set console output type, candidates: ${COMPLETION-CANDIDATES}, default: ${DEFAULT-VALUE}")
  @SuppressWarnings("NullAway.Init") // initialized by picocli
  private ConsoleOutput consoleOutput;
```